### ISSUE TITLE
unbreakeble block 1.12.2 revelations 1.3.0

### ISSUE THREAD

**Issue by unstablecraft** [Thutmose/Thut#92]
![image](https://user-images.githubusercontent.com/33659444/35182632-f2521bf8-fdd8-11e7-9e17-690be374c503.png)

for some reason we cant remove this block in anyway

**Comment by Thutmose** [comment]
Whoever made that elevator needs to shift right click it with a stick, or I think any player in creative mode

**Comment by unstablecraft** [comment]
any player in creative isnt working we tried that but will try the shift click method and report back :D

**Comment by sathelate** [comment]
if you have other mods, you might need a wrench if thut found it in FML start ;) 
i needed to make myself a wrench from extra utilities, then sneak right clicking just put the block to solid... you remove this one block, and recreate the elevator process :) you should be golden then :)

**Comment by Thutmose** [comment]
if a wrench worked, then a stick should work as well, it is supposed to check wrenches and sticks

**Comment by sathelate** [comment]
You are absolutely right thut :D my bad, yep i can also "unmount" the elevator using regular stick ;)

Hope OP figured it out then ^_^
